setup.py

```diff
@@ -12,6 +12,7 @@
     packages=find_packages(),
     install_requires=[
         "tqdm",
+        "httpx==0.27.2",
         "anthropic",
```

Comment:
can we remove our dependency to anthropic now?

Reply:
Yes!

Reply:
Added a new commit!

Reply:
Was the new commit supposed to remove `anthropic` from `setup.py`? I don't see that change here.

Reply:
Just added a new commit to address this!